instituto-de-investigaciones-juridicas-unam.csl

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="es-MX">
+  <info>
+    <title>Instituto de Investigaciones Jurídicas UNAM (es-MX)</title>
+    <title-short>IIJ-UNAM</title-short>
+    <id>http://www.zotero.org/styles/instituto-de-investigaciones-juridicas-unam</id>
+    <link href="http://www.zotero.org/styles/instituto-de-investigaciones-juridicas-unam" rel="self"/>
+    <link href="https://revistas.juridicas.unam.mx/index.php/derecho-internacional/libraryFiles/downloadPublic/34" rel="documentation"/>
```

Comment:
Please add a link to the style that was used as a template as per our style requirements. https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md